projects/igniteui-angular/src/lib/tabs/tabs-routing-view-components.ts

```diff
@@ -0,0 +1,29 @@
+import { Component, NgModule } from '@angular/core';
+
+@Component({
+    template: `This is a content from view component # 1`
+})
+export class TabsRoutingView1Component {
```

Comment:
Update views' content.

Reply:
These templates are used for the tests only. It is ok to have short content. Renaming the files however to have "spec" in their names.